cmd/openshift/operator/kodata/operator-namespace.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
```

Comment:
This is already available in pipeline release.yaml Why we need this twice? Is it because of the prereconcile thing

Reply:
Yes, this code is run before and it fails because the namespace does not exist yet 🤦🏼